### PR TITLE
chore: [Multi-domain] Update error messages based on feedback.

### PR DESCRIPTION
**Pull request by mjhenkes**
- Closes 

### User facing changelog

n/a

### Additional details

Added some updates to the error messages based on feedback

cross_origin_load_timed_out:
* added cy.origin docs url
* removed redundant page did not load message

cross_origin_command:
* removed colons
* added further explanation of likely causes.

### How has the user experience changed?

When a page load times out when a cross origin request exists
Screen Shot 2022-04-08 at 8 34 59 AM

When a command times out and the aut origin does not match the command origin
Screen Shot 2022-04-08 at 8 36 45 AM

### PR Tasks

- [ ] Have tests been added/updated?
- [ ] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
- [ ] Have new configuration options been added to the [`cypress.schema.json`](https://github.com/cypress-io/cypress/blob/develop/cli/schema/cypress.schema.json)?

### CODE REVIEW

File: packages/driver/src/cypress/error_messages.ts [hunk 1]
```diff
@@ -895,7 +895,10 @@ export default {
       return `Timed out retrying after ${ms}ms: `
     },
     cross_origin_command ({ commandOrigin, autOrigin }) {
-      return `The command was expected to run against origin: \`${commandOrigin }\` but the application is at origin: \`${autOrigin}\`.`
+      return stripIndent`\
+        The command was expected to run against origin \`${commandOrigin }\` but the application is at origin \`${autOrigin}\`.
+
+        This commonly happens when you have either not navigated to the expected origin or have navigated away unexpected.`
```
Comment (emilyrohrbough) [inline packages/driver/src/cypress/error_messages.ts:901]:
```suggestion
        This commonly happens when you have either not navigated to the expected origin or have navigated away unexpectedly.`
```

File: packages/driver/src/cypress/error_messages.ts [hunk 2]
```diff
@@ -989,29 +992,29 @@ export default {
...
+          cy.origin('${crossOriginUrl.originPolicy}', () => {
+            <commands targeting ${crossOriginUrl.origin} go here>
+          })
+          \`\`\`
 
-        If the cross origin request was an intermediary state, you can try increasing the \`pageLoadTimeout\` value in ${formatConfigFile(configFile)} to wait longer.
+          If the cross origin request was an intermediary state, you can try increasing the \`pageLoadTimeout\` value in ${formatConfigFile(configFile)} to wait longer.
 
-        Browsers will not fire the \`load\` event until all stylesheets and scripts are done downloading.
+          Browsers will not fire the \`load\` event until all stylesheets and scripts are done downloading.
```
Comment (emilyrohrbough) [inline packages/driver/src/cypress/error_messages.ts:1013]: was this meant to be another reason an increased timeout might be needed?

File: packages/driver/src/cypress/error_messages.ts [hunk 3]
```diff
@@ -989,29 +992,29 @@ export default {
...
+          Timed out after waiting \`${ms}ms\` for your remote page to load on origin(s):
 
-        Your page did not fire its \`load\` event within \`${ms}ms\`.
+          - ${originPolicies.map((originPolicy) => `\`${originPolicy}\``).join('\n       -')}
 
-        A cross origin request for \`${crossOriginUrl.href}\` was detected.
+          A cross origin request for \`${crossOriginUrl.href}\` was detected.
 
-        A command that triggers cross origin navigation must be immediately followed by a ${cmd('origin')} command:
+          A command that triggers cross origin navigation must be immediately followed by a ${cmd('origin')} command:
```
Comment (emilyrohrbough) [inline packages/driver/src/cypress/error_messages.ts:1003]:
```suggestion
          A command that triggers cross origin navigation must be immediately followed by a ${cmd('origin')} command to switch origins:
```

File: packages/driver/src/cypress/error_messages.ts (hunk 2 above)
Comment (mjhenkes) [reply packages/driver/src/cypress/error_messages.ts:1013]: I think so? mostly this message was cribbed from here, with added cy.origin stuff added:
https://github.com/cypress-io/cypress/blob/feature-multidomain/packages/driver/src/cypress/error_messages.ts#L973-L984

File: packages/driver/src/cypress/error_messages.ts (hunk 1 above)
Comment (mjhenkes) [reply packages/driver/src/cypress/error_messages.ts]: whoops, updated

**Review by emilyrohrbough — APPROVED** [review]

File: packages/driver/src/cypress/error_messages.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (mjhenkes) [reply packages/driver/src/cypress/error_messages.ts]: I see instances of both in our code, but i've changed this message to use hyphens.

File: packages/driver/src/cypress/error_messages.ts (hunk 2 above)
Comment (mschile) [reply packages/driver/src/cypress/error_messages.ts:1013]: I wouldn't be opposed to removing this line and the next.

**Review by mschile — APPROVED** [review]

**Pull request merged**